train_and_eval/eval/calibration_tools.py

```diff
@@ -0,0 +1,202 @@
+import numpy as np
```

Comment:
header comment for which part of paper this code was used for (and for all files in eval/ dir)

Reply:
yes.

Reply:
This is actually never used in the main paper, however it provides a provision to evaluate on ImageNet-O